geowebcache/rest/src/main/java/org/geowebcache/rest/service/FormService.java

```diff
@@ -182,28 +182,31 @@
             if (tl == null) {
                 throw new RestException("No layer specified", HttpStatus.BAD_REQUEST);
             }
-            handleDoSeedPost(params, tl);
+            return handleDoSeedPost(params, tl);
         } else {
             throw new RestException(
                     "Unknown or malformed request. Please try again, somtimes the form "
                             + "is not properly received. This frequently happens on the first POST "
                             + "after a restart. The POST was to " + request.getRequestURI(),
                     HttpStatus.BAD_REQUEST);
         }
-        return null;
     }
 
     private ResponseEntity<?> handleDoSeedPost(Map<String, String> form, TileLayer tl) throws RestException,
             GeoWebCacheException {
+        String gridSetId = form.get("gridSetId");
+
         BoundingBox bounds = null;
 
-        if (form.get("minX") != null) {
+        if (!form.get("minX").equalsIgnoreCase("")) {
```

Comment:
`form.get("minX") != null && !form.get("minX").equalsIgnoreCase("")` is probably safer